### PR TITLE
make copyright year dynamic

### PR DESCRIPTION
**Pull request by pdurbin**
Instead of 2017 (two years ago), let's make the year dynamic.

I didn't test this but it worked at https://github.com/IQSS/dataverse/commit/d8f1b98a6776c25b79312e44f1e6ea7baa24a1e0 from what I understand. 😄

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -59,7 +60,7 @@ def setup(app):
 
 # General information about the project.
 project = 'Site Reliability Guide for mybinder.org'
-copyright = '2017, Binder Team'
+copyright = '%d, Binder Team' % datetime.now().year
```
Comment (betatim) [inline docs/source/conf.py]:
```suggestion
copyright = '2017 - %d, Binder Team' % datetime.now().year
```

IANAL. This way we preserve the starting date which is nice

Comment (choldgraf) [reply docs/source/conf.py]: I tried committing this but it wouldn't let me :-/

**Comment by choldgraf** [comment]
@pdurbin I believe that we don't have permission to push to your branch, can you decide to accept @betatim's suggestion above (or not) so we can get this in? :-)

**Comment by pdurbin** [comment]
@choldgraf done! I added 469fb34 by @betatim into my branch. Thanks for the heads up.

**Comment by choldgraf** [comment]
wooo, thanks!

**Pull request merged**